Fixes these warnings:
configure.ac:648: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in
body

Index: rezound-0.12.3beta/config/m4/cxx-lib.m4
===================================================================
--- rezound-0.12.3beta.orig/config/m4/cxx-lib.m4
+++ rezound-0.12.3beta/config/m4/cxx-lib.m4
@@ -126,10 +126,10 @@ AC_DEFUN([ajv_CXX_CHECK_LIB],
 		
 		builddir=`pwd`  # I'm assuming that the pwd will always be what $(top_builddir) will be set to in AC_OUTPUT
 
-		[AC_LINK_IFELSE([
+		[AC_LINK_IFELSE([AC_LANG_SOURCE([
 				#include <$3>
 				int main() { $2 xxx; return 0; }
-			],
+			])],
 			[ # worked
 				AC_MSG_RESULT(yes)
 				AC_DEFINE_UNQUOTED(AS_TR_CPP(HAVE_LIB$1))
